src/components/plots/plot.ts

```diff
@@ -267,6 +278,8 @@ module Plottable {
     protected _updateExtents() {
       Object.keys(this._projections).forEach((attr: string) => { this._updateExtentsForAttr(attr); });
       this._scales().forEach((scale) => scale._autoDomainIfAutomaticMode());
+      this._propertyExtents.forEach((property) => this._updateExtentsForProperty(property));
+      this._propertyScales().forEach((scale) => scale._autoDomainIfAutomaticMode());
```

Comment:
This seems silly. `_scales()` should just return all `Scale`s, including those associated with properties.

Splitting them also causes a problem where if a scale is used for both a property and an attribute you'll get two `autodomain()` calls, the first of which is wrong.